packages/compass-generative-ai/scripts/ai-accuracy-tests.ts

```diff
@@ -211,7 +213,8 @@ const generateMQL = async ({
       schema: schema,
       collectionName,
       databaseName,
-      sampleDocuments: USE_SAMPLE_DOCS ? sample : undefined,
+      sampleDocuments:
+        includeSampleDocuments || USE_SAMPLE_DOCS ? sample : undefined,
```

Comment:
Can we rename `USE_SAMPLE_DOCS` to `AI_TESTS_USE_SAMPLE_DOCS` so it is more clear what it does?